### PR TITLE
Splitted default queue to import/export, fixed django-rq admin page

### PR DESCRIPTION
**Pull request by azhavoro**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat/apps/engine/backup.py [hunk 1]
```diff
@@ -762,8 +762,8 @@ def export(db_instance, request):
         field_name=StorageType.TARGET
     )
 
-    queue = django_rq.get_queue("default")
-    rq_id = "/api/{}s/{}/backup".format(filename_prefix, db_instance.pk)
+    queue = django_rq.get_queue(queue_name)
+    rq_id = f"api-{filename_prefix}s-{db_instance.pk}/backup"
```
Comment (nmanovic) [inline cvat/apps/engine/backup.py:766]: -backup?

File: cvat/apps/engine/backup.py [hunk 2]
```diff
@@ -905,11 +904,11 @@ def _import(importer, request, rq_id, Serializer, file_field_name, location_conf
 def get_backup_dirname():
     return settings.TMP_FILES_ROOT
 
-def import_project(request, filename=None):
+def import_project(request, queue_name, filename=None):
     if 'rq_id' in request.data:
         rq_id = request.data['rq_id']
     else:
-        rq_id = "{}@/api/projects/{}/import".format(request.user, uuid.uuid4())
+        rq_id = f"{request.user}$-api-projects-{uuid.uuid4()}-import"
```
Comment (nmanovic) [inline cvat/apps/engine/backup.py:911]: Is `$` typo?

File: cvat/apps/engine/backup.py [hunk 3]
```diff
@@ -918,21 +917,24 @@ def import_project(request, filename=None):
...
         filename=filename
     )
 
-def import_task(request, filename=None):
+def import_task(request, queue_name, filename=None):
     if 'rq_id' in request.data:
         rq_id = request.data['rq_id']
     else:
-        rq_id = "{}@/api/tasks/{}/import".format(request.user, uuid.uuid4())
+        rq_id = f"{request.user}$-api-tasks-{uuid.uuid4()}-import"
```
Comment (nmanovic) [inline cvat/apps/engine/backup.py:937]: `@` meant `in`. I see no value in `$` here

File: cvat/apps/engine/backup.py (hunk 2 above)
Comment (nmanovic) [reply cvat/apps/engine/backup.py:911]: I will recommend to think about more readable names: `import~project.{uuid.uuid4()}-by-{request.user.username}`

File: cvat/apps/engine/backup.py (hunk 3 above)
Comment (nmanovic) [reply cvat/apps/engine/backup.py:937]: `import~task.{uuid.uuid4()}-by-{request.user.username}`

File: cvat/apps/engine/mixins.py [hunk 4]
```diff
@@ -259,7 +259,7 @@ def export_annotations(self, request, pk, db_obj, export_func, callback, get_dat
             field_name=StorageType.TARGET,
         )
 
-        rq_id = "/api/{}/{}/annotations/{}".format(self._object.__class__.__name__.lower(), pk, format_name)
+        rq_id = f"api-{self._object.__class__.__name__.lower()}-{pk}-annotations-{format_name}"
```
Comment (nmanovic) [inline cvat/apps/engine/mixins.py:262]: `export~annotations-for-{self._object.__class__.__name__.lower()}.id{pk}-in-{format_name}-format`

File: cvat/apps/engine/task.py [hunk 5]
```diff
@@ -40,9 +40,9 @@
 
 def create(tid, data):
     """Schedule the task"""
-    q = django_rq.get_queue('default')
+    q = django_rq.get_queue(settings.CVAT_QUEUES.IMPORT_DATA.value)
     q.enqueue_call(func=_create_thread, args=(tid, data),
-        job_id="/api/tasks/{}".format(tid))
+        job_id=f"api-tasks-{tid}")
```
Comment (nmanovic) [inline cvat/apps/engine/task.py:45]: `create~task.id{tid}-by-{username}`

File: supervisord/worker.annotation.conf [hunk 6]
```diff
@@ -24,15 +24,8 @@ autorestart=true
 
 [program:rqworker_low]
```
Comment (nmanovic) [inline supervisord/worker.annotation.conf:25]: rqworker_annotation for consistency?

File: supervisord/worker.export.conf [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+loglevel=debug              ; info, debug, warn, trace
+pidfile=/tmp/supervisord/supervisord.pid ; pidfile location
+childlogdir=%(ENV_HOME)s/logs/            ; where child log files will live
+
+[program:ssh-agent]
+command=bash -c "rm /tmp/ssh-agent.sock -f && /usr/bin/ssh-agent -d -a /tmp/ssh-agent.sock"
+priority=1
+autorestart=true
+
+[program:rqworker_default]
```
Comment (nmanovic) [inline supervisord/worker.export.conf:25]: rqworker_export?

File: supervisord/worker.export.conf [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+command=%(ENV_HOME)s/wait-for-it.sh %(ENV_CVAT_REDIS_HOST)s:6379 -t 0 -- bash -ic " \
+    exec python3 %(ENV_HOME)s/manage.py rqworker -v 3 export \
+        --worker-class cvat.rqworker.DefaultWorker \
+    "
+environment=SSH_AUTH_SOCK="/tmp/ssh-agent.sock"
+numprocs=%(ENV_NUMPROCS)s
+process_name=rqworker_default_%(process_num)s
```
Comment (nmanovic) [inline supervisord/worker.export.conf:32]: rqworker_export_?

**Review by nmanovic — APPROVED** [review]

**Comment by StellaASchlotter** [comment]
@azhavoro there is a reference to cvat_worker_default in the docs share path example. See https://opencv.github.io/cvat/docs/administration/basics/installation/#share-path

How should I adjust that in my override file? Do I use cvat_worker_import or something else?

**Pull request merged**